educational-core/src/com/jetbrains/edu/learning/codeforces/courseFormat/CodeforcesCourse.kt

```diff
@@ -39,7 +39,7 @@ class CodeforcesCourse : Course {
   override fun isViewAsEducatorEnabled(): Boolean = false
 
   fun getContestUrl(): String = getContestURLFromID(id)
-  fun getSubmissionUrl(): String = "${getContestUrl()}/${CODEFORCES_SUBMIT}?locale=$languageCode"
+  fun getSubmissionUrl(task: CodeforcesTask? = null): String = "${getContestUrl()}/${CODEFORCES_SUBMIT}?locale=$languageCode${if (task == null) "" else "&submittedProblemIndex=" + task.name.substringBefore(".")}"
```

Comment:
Maybe move this method to `CodeforcesTask`? I think now it's more suitable there.

Reply:
I thought about it, but did not want to change too much in the existing code. As it's ok I'll move it to `CodeforcesTask` so concert about nullable will be resolved.

Reply:
With your change, we wouldn't need it here, so I don't see why we should stay it here. Don't worry about changing too much, it's fine :)